src/index.html

```diff
@@ -4,8 +4,26 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wbpack Exercise</title>
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <h1>Hello webpack!</h1>
+    <h1>Leaderboard</h1>
+    <main class="main">
+        <section class="scoresSection">
+            <div class="scoresTitle">
+                <h2>Recent Scores</h2>
+                <button id="refreshButton">Refresh</button>
+            </div>
+            <div class="scores" id="scoresList">
+            </div>
+        </section>
+        <section class="addScoreSection">
+            <h2>Add your Score</h2>
```

Comment:
**[Optional]**

- [ ] It would be nice if you would use the `form` tag instead of the `section` tag because it is your form and the form has very different purposes and methods from the section. Please take it into consideration :blush: